docs/configuration-guide.md

```diff
@@ -143,7 +143,8 @@ restrictions are applied for label names:
 - Names cannot start with `_`, but it can be present in other parts of the name.
 - Names only accept alphabet letters, numbers, dots, and dashes.
 
-The policy for values are more flexible, but you can not use exclamation (`!`), whitespaces (` `)  and asterisk (`*`), 
+The policy for values are more flexible, but you can not use exclamation (`!`), whitespaces (` `), single quotes (`'`),
```

Comment:
"The policy for values is more flexible" and
"exclamation" -> "exclamation marks"

Reply:
It was changed in the last commit!